storage_service/locations/models/space.py

```diff
@@ -141,7 +141,7 @@ class Space(models.Model):
     OBJECT_STORAGE = {DATAVERSE, DSPACE, DURACLOUD, SWIFT, S3}
     ACCESS_PROTOCOL_CHOICES = (
         (ARKIVUM, _l('Arkivum')),
-        (DATAVERSE, _l('Dataverse')),
+        (DATAVERSE, _l('Dataverse (Beta)')),
```

Comment:
This produces a new database migration which I think is not ideal. You may have noticed that Travis is complaining because your PR does not include the new migration.